sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/splittabledofn/Sizes.java

```diff
@@ -1,54 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.beam.sdk.transforms.splittabledofn;
-
-import org.apache.beam.sdk.annotations.Experimental;
-import org.apache.beam.sdk.annotations.Experimental.Kind;
-
-/** Definitions and convenience methods for reporting sizes for SplittableDoFns. */
-@Experimental(Kind.SPLITTABLE_DO_FN)
-public final class Sizes {
-  /**
-   * {@link RestrictionTracker}s which can provide a size should implement this interface.
-   * Implementations that do not implement this interface will be assumed to have an equivalent
-   * size.
-   */
-  public interface HasSize {
```

Comment:
I though we still want to keep this and make `HasProgress` as a fallback?

Reply:
The intent was to remove using size as a progress signal. Which means that all restriction trackers need to implement the slightly more complicated HasProgress. Since that is the case, there is no need to keep HasSize around as a fallback since we can use the work remaining as the default size if there is no `@GetSize` override.